crates/cargo-bsp/tests/features_extension_integration_test.rs

```diff
@@ -0,0 +1,215 @@
+//! Integration tests for Cargo extension of BSP. The test in this module switches te environment,
```

Comment:
```suggestion
//! Integration tests for Cargo extension of BSP. The test in this module switches the environment,
```
Trochę nie rozumiem co to znaczy still że "switches the environment"

Reply:
Chodzi o to, że zmienia się folder, w którym jest wykonywany kod, tylko nie wiem na ile potrzebny jest ten komentarz